### ISSUE TITLE
About the LGA function.

### ISSUE THREAD

**Issue by yiying66** [feihuzhang/GANet#55]
Hi, 
    thank you for your sharing.
    I have some problems when training the model. At first, I found the model does not converge. And then  I found that it is because the outputs of the LGA function are always be zero. I made sure the input was ok.  I only replace the `sync_bn` with `torch.nn.BatchNorm`.
   Looking forward to your reply.

**Comment by chenglong0313** [comment]
how do you solve this problem？  I seem to meet it too. and my snyc bn is always running wrong.
